crm_department/README.rst

```diff
@@ -0,0 +1,76 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+CRM with Department Categorization
+===========
+
+
+
+Add the department on Sales Team (with related field on Lead &amp; Opportunities)
+as well as the related filter and button in the search form.
+
+
+
+Installation
+============
+
+To install this module, you need to:
+
+* do this ...
```

Comment:
Simply remove those part if not filled I believe this module is quite simple.

So we don't need Installation, Configuration, Usage and Know issues / Roadmap